app/controllers/users_controller.rb

```diff
@@ -2,7 +2,7 @@
 class UsersController < ApplicationController
   before_action :retrieve_patients, only: [:add_patient, :remove_patient]
   before_action :confirm_admin_user, only: [:new, :index, :update]
-  before_action :find_user, only: [:update, :edit, :destroy, :reset_password]
+  before_action :find_user, only: [:update, :edit, :destroy, :reset_password, :clear_call_list]
```

Comment:
No need to do the `before_action` on `:clear_call_list`, since you aren't doing a user lookup on id.